src/main/scala/org/globalforestwatch/layers/Layer.scala

```diff
@@ -147,7 +148,17 @@ trait RequiredLayer extends Layer {
     // Removes the expected 404 errors from console log
     val s3uri = new AmazonS3URI(uri)
     try {
-      if (!s3Client.doesObjectExist(s3uri.getBucket, s3uri.getKey)) {
+      val keyExists =
+        try {
+          val headRequest = HeadObjectRequest.builder().bucket(s3uri.getBucket).key(s3uri.getKey).build()
+          s3Client.headObject(headRequest)
+          true
+        } catch {
+          case e: NoSuchKeyException => false
+          case _: Throwable => throw new Exception(f"Unexpected exception on key ${s3uri.getKey}")
```

Comment:
Codacy found an issue: [Fatal exception is being caught](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)